app/controllers/qualifications/feedbacks_controller.rb

```diff
@@ -0,0 +1,29 @@
+class FeedbacksController < ApplicationController
+  skip_before_action :authenticate_dsi_user!
+  skip_before_action :handle_expired_session!
```

Comment:
The auth methods that need skipping are different in AYTQ - see the `QualificationsInterfaceController`.